content/information-distribution/node.md

```diff
@@ -54,14 +57,108 @@ the Matrix room to join.
+final parameters to reach the node (i.e. consider the node unreachable if those
+parameters don't work).
+7. If step 5 fails, consider the node's server name and the `8448` port as the
+final parameters to reach the node (i.e. consider the node unreachable if those
+parameters don't work).
+
+This should result in making the network traffic generated by Informo client
+implementations more difficult to detect and filter on.
+
+If a client implementation's maintainer(s) decide not to use port 443, their
+client implementation **must** implement the following workflow:
+
+1. Try retrieving a Matrix `SRV` DNS record from the node's server name.
+2. If the DNS record exists, consider the FQDN and port extracted from the
+DNS record as the final parameters to reach the node (i.e. consider the node
+unreachable if those parameters don't work).
+3. If the DNS record doesn't exist, consider the node's server name and the
+`8448` port as the final parameters to reach the node (i.e. consider the node
+unreachable if those parameters don't work).
+
+{{% notice tip %}}
+Web-based client implementations **can** use the [DNS over
+HTTPS](https://tools.ietf.org/html/rfc8484) standard to perform DNS queries over
+HTTPS instead of standard DNS queries which they might not be able to perform.
 {{% /notice %}}
+
+In the event of an entry node that's currently being used becoming unreachable,
+clients implementations **should** use the map generated from the room aliases
+list to provide its user(s) with alternative entry nodes and aliases to use.
+Client implementations **can** use received pings in order to filter out the
+inactive nodes from this list.
+
+### Joining a federation
+
+Once both an entry node and a room alias have been selected as the one to use in
+order to join a federation, whether this selection originated from an explicit
+action by a user or, either partially or entirely, from a client
+implementation's inner mechanisms, client implementations **must** send a [join
+request](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#post-matrix-client-r0-join-roomidoralias)
+to the selected node using the selected alias.
+
+## Other types of nodes
+
+Nodes that aren't entry nodes (e.g. a node with no guest access or no alias
+pointing towards the Matrix room) are named "relay nodes". Although they cannot
+be used by client implementations in order to retrieve articles from the Matrix
+room, they are still useful as they can relay articles to other nodes that are
+out of reach from the articles' emitter(s). For example, if a node `A` can't
+reach a node `B` but can reach a node `C`, even if `C` isn't an entry node,
+`A`'s information will eventually manage to reach `B` via `C` thanks to pings.
+This is done through [Matrix's `get_missing_events` federation
+endpoint](https://matrix.org/docs/spec/server_server/unstable.html#post-matrix-federation-v1-get-missing-events-roomid).
```

Comment:
Remove this if you add the thing line 38

Reply:
Fixed in https://github.com/Informo/specs/pull/2/commits/1560426c483f08b99843f0ea489063d9522741dc.